### ISSUE TITLE
When I start a range, it attempts to open the completions

### ISSUE THREAD

**Issue by onionhammer** [barcharcraz/VisualNimrod#3]
I type in: for i in 0.. 5:

When I first tried, on the first  "." I got an error box. The second time I got a list starting with system.ordinal.

**Comment by barcharcraz** [comment]
I am not sure what the desired behavior here is, you can use the method syntax on ordinals. Also figuring out what is an ordinal in order to include enums in an exception requires actually parsing the code, or using IDETools a second time, which would slow things down even more. I may try and stop idetools after the second dot is pressed.

**Comment by onionhammer** [comment]
Yeah.. I think thats the desired behavior IMO - if I hit "." and the box pops up, then I hit "." again it should dismiss the box and put in another dot.

**Comment by barcharcraz** [comment]
allright, I turned on async completions and added checks to return "" if the IDE asks for definitions off the end of the declarations array. This seems to help the errors and the issues with `..`.
